src/Discovery/aggMDSUtils.jsx

```diff
@@ -18,7 +18,8 @@ const retrieveCommonsInfo = async (commonsName) => {
  * @param {*} tags
  * @returns array with duplicates removed
  */
-const getUniqueTags = ((tags) => tags.filter((v, i, a) => a.findIndex((t) => (t.category === v.category && t.name === v.name)) === i));
+const getUniqueTags = ((tags) => tags.filter((v, i, a) => a.findIndex((t) => (
+  typeof t.name === 'string' && t.name.length > 0 && t.category === v.category && t.name === v.name)) === i));
```

Comment:
```suggestion
  t.name?.length > 0 && t.category === v.category && t.name === v.name)) === i));
```
can it be done with that?

Reply:
Nice! Yes, that looks better.